staging/src/k8s.io/apiserver/pkg/util/flowcontrol/apf_controller.go

```diff
@@ -1049,6 +1063,30 @@ func (cfgCtlr *configController) maybeReapReadLocked(plName string, plState *pri
 	cfgCtlr.configQueue.Add(0)
 }
 
+// PriorityLevelMaxSeats returns the maximum seats that a request should occupy
+// for a given priority level. See priorityLevelMaxSeatsLocked for more details.
+func (cfgCtlr *configController) PriorityLevelMaxSeats(plName string) uint64 {
```

Comment:
It is unfortunate that this change makes the locking discipline require global reasoning over all the code rather than the designed local reasoning independently in each method.

Reply:
https://github.com/kubernetes/kubernetes/pull/118601#discussion_r1262122200